README.md

```diff
@@ -1,6 +1,24 @@
 # Baby Project Deployment
-This repository serves as a form of documentation and deployment mechanism 
-through github actions for how we deploy the Baby Project to a server along with
-what security measures we have in place for securing the app and the server(s).
+We use a micro repo structure for a few reasons, but the main one is that we can make use of
+GitHub pages for each repo separately. That way we can deploy docs, test coverage or even a full webapp without extra
```

Comment:
Same opinion as Otto, I think it's better to avoid abbreviations when not needed. 😄

```suggestion
GitHub pages for each repo separately. That way we can deploy documents/our documentation, test coverage or even a full webapp without extra
```